src/relay/environment.js

```diff
@@ -41,6 +42,12 @@ const buildFetchQuery = (
       }),
     });
     const json = await response.json();
+    if (json.errors && json.errors.length > 0) {
+      const errorMsg = json.errors.some(e => e._proxy.statusCode === 403)
+        ? ERROR_FORBIDDEN
+        : '';
+      throw new Error(errorMsg);
```

Comment:
- We should allow partial errors
- throwing error with empty message doesn't make much sense